arm-keyvault/2015-06-01/swagger/keyvault.json

```diff
@@ -0,0 +1,459 @@
+        },
+        "permissions": {
+          "$ref": "#/definitions/Permissions",
+          "description": "Permissions the identity has for keys and secrets"
+        }
+      }
+    },
+    "Permissions": {
+      "properties": {
+        "keys": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Permissions to keys"
+        },
+        "secrets": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Permissions to secrets"
+        }
+      }
+    },
+    "VaultProperties": {
+      "properties": {
+        "vaultUri": {
+          "type": "string",
+          "description": "The URI of the vault for performing operations on keys and secrets."
+        },
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "The Azure Active Directory tenant ID that should be used for authenticating requests to the key vault."
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "SKU details"
+        },
+        "accessPolicies": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AccessPolicyEntry"
+          },
+          "maxItems": 16,
+          "description": "An array of 0 to 16 identities that have access to the key vault. All identities in the array must use the same tenant ID as the key vault�s tenant ID."
+        },
+        "enabledForDeployment": {
+          "type": "boolean",
+          "description": "Property to specify whether Azure Virtual Machines are permitted to retrieve certificates stored as secrets from the key vault."
+        },
+        "enabledForDiskEncryption": {
+          "type": "boolean",
+          "description": "Property to specify whether Azure Disk Encryption is permitted to retrieve secrets from the vault and unwrap keys."
+        },
+        "enabledForTemplateDeployment": {
+          "type": "boolean",
+          "description": "Property to specify whether Azure Resource Manager is permitted to retrieve secrets from the key vault."
+        }
+      },
+      "required": [
+        "tenantId",
+        "sku",
+        "accessPolicies"
+      ],
+      "description": "Properties of the vault"
+    },
+    "VaultCreateOrUpdateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The supported Azure location where the key vault should be created."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The tags that will be assigned to the key vault. "
+        },
+        "properties": {
+          "$ref": "#/definitions/VaultProperties",
+          "description": "Properties of the vault"
```

Comment:
Same as above, could you move this description to the VaultProperties definition?